backend/src/main/java/com/yigongil/backend/domain/studymember/StudyMember.java

```diff
@@ -54,6 +54,14 @@ public boolean isNotApplicant() {
         return role != Role.APPLICANT;
     }
 
+    public boolean isStudyEnd() {
+        return !studyResult.equals(StudyResult.NONE);
```

Comment:
p3: 저는 enum의 경우 `==` 또는 `!=`으로 비교하는 게 더 보기 좋더라고요. 어떻게 생각하시나요?

```suggestion
        return studyResult != StudyResult.NONE;
```